labs/lab-1-html-and-css/desktop-stylesheet.css

```diff
@@ -0,0 +1,131 @@
+/*
+Author: Vikram Badarinath
+Created: 01/23/2018
+*/
+
+body {
+  color: #69696a;
+  background-color: #f9fafc;
+  font-family: Open Sans;
+  font-weight: 800;
+}
+
+/* top of the page*/
+
+.menu {
+  flex: 1;
+  display: flex;
+  flex-direction: row;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.title {
+  flex: 1;
+  color: #000;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 500px;
+}
+
+a:link, a:visited {
+  color: inherit;
+  text-decoration: none;
+}
+
+.tab {
+  padding: 5px;
+  border: solid #f9fafc;
+  border-radius: 100px;
+}
+
+.special {
```

Comment:
Nice noticing that you can share these styles between scroll down and hire us!

Maybe there's a better name for that class, though: `black-button` or something that you'll recognize and remember in a month.